locales/en/index.yml

```diff
@@ -1412,6 +1412,10 @@ wallet:
       manually enter the data (Notice Code and Payee Fiscal Code).
     setManually: Enter manually
     enroll_cta: To scan the QR code, IO needs your permission to use your phone camera. This can be done from the 'Settings' menu of your device.
+    readStorageDisclosure:
+      title: Allow gallery access
+      message: You’ll be able to upload screenshots and payment notices.
+      cta: Scegli
```

Comment:
Since this key is not used, can we remove it?

Reply:
Sure I totally forgot to remove it
https://github.com/pagopa/io-app/pull/3563/commits/f77c5dc4ef4e16f6bb1f2947db2db41665db28a9